validator_manager/src/move_validators.rs

```diff
@@ -0,0 +1,1261 @@
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(DEST_VC_URL_FLAG)
+                .long(DEST_VC_URL_FLAG)
+                .value_name("HTTP_ADDRESS")
+                .help(
+                    "A HTTP(S) address of a validator client using the keymanager-API. \
+                    This validator client is the \"destination\" and will have new validators \
+                    added as they are removed from the \"source\" validator client.",
+                )
+                .required(true)
+                .requires(DEST_VC_TOKEN_FLAG)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(DEST_VC_TOKEN_FLAG)
+                .long(DEST_VC_TOKEN_FLAG)
+                .value_name("PATH")
+                .help("The file containing a token required by the destination validator client.")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(VALIDATORS_FLAG)
+                .long(VALIDATORS_FLAG)
+                .value_name("STRING")
+                .help(
+                    "The validators to be moved. Either a list of 0x-prefixed \
+                    validator pubkeys, an integer count of validators or the \
+                    keyword \"all\".",
+                )
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(COUNT_FLAG)
+                .long(COUNT_FLAG)
+                .value_name("VALIDATOR_COUNT")
+                .help("The number of validators to move.")
+                .conflicts_with(VALIDATORS_FLAG)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(GAS_LIMIT_FLAG)
+                .long(GAS_LIMIT_FLAG)
+                .value_name("UINT64")
+                .help(
+                    "All created validators will use this gas limit. It is recommended \
+                    to leave this as the default value by not specifying this flag.",
+                )
+                .required(false)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(FEE_RECIPIENT_FLAG)
+                .long(FEE_RECIPIENT_FLAG)
+                .value_name("ETH1_ADDRESS")
+                .help(
+                    "All created validators will use this value for the suggested \
+                    fee recipient. Omit this flag to use the default value from the VC.",
+                )
+                .required(false)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name(BUILDER_PROPOSALS_FLAG)
+                .long(BUILDER_PROPOSALS_FLAG)
+                .help(
+                    "When provided, all created validators will attempt to create \
+                    blocks via builder rather than the local EL.",
+                )
+                .required(false)
+                .possible_values(&["true", "false"])
+                .takes_value(true),
```

Comment:
```suggestion
                .required(false),
```

Not sure if this flag should take value, because we only check whether it's present here:

https://github.com/sigp/lighthouse/blob/09bf5ab4314d6ece806fe8aa368d1de0191dd256/validator_client/src/config.rs#L360-L362

Reply:
oh sorry my bad, please ignore this, I was looking at the VC config!